masonry_core/src/passes/layout.rs

```diff
@@ -212,7 +219,7 @@ pub(crate) fn place_widget(child_state: &mut WidgetState, origin: Point) {
 
     // TODO - We may want to invalidate in other cases as well
     if origin != child_state.origin {
-        child_state.transform_changed = true;
+        changed_properties.mark_as_changed::<Transform>();
```

Comment:
Not sure whether it's the right thing to change this. Alternatively we could leave the `transform_changed` flag within `WidgetState` additionally, and check both in the compose pass (or set `transform_changed` as well in `WidgetMut::property_changed`).

Reply:
I think this alternative is the better way around. My understanding is that the semantics of `prop_has_changed` are very clear, which is that they mark whether the transform has changed during the lifetime of this `WidgetMut`, right?